src/services/hotstar/HotstarParser.ts

```diff
@@ -5,35 +5,36 @@ import { EpisodeItem, MovieItem } from '@models/Item';
 class _HotstarParser extends ScrobbleParser {
 	constructor() {
 		super(HotstarApi, {
-			watchingUrlRegex: /\/(?:movies|tv)\/(?<id>.+)/,
+			watchingUrlRegex: /\/(?:movies|shows)\/\D+(?<id>\d.+)\//,
```

Comment:
I don't think this is correct for episodes.

Using the URLs from your screenshots:

- `hotstar.com/id/shows/dr-stone/1260117151/dr-stone/1260117832/watch?filters=content_type%3Depisode` results in `1260117151/dr-stone/1260117832` 
- `hotstar.com/id/movies/sri-asih/1260133861/watch?fiIters=content_type%3Dmovie` results in `1260133861`

I think for episodes it should result in `1260117832`.

So a better regex would probably be: `/(?:movies|shows)\/.+\/(?<id>\d+)\//`

Reply:
Thank you for correction. The regex was updated